src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+
+  --link-color: #00acdc;
+  --header-height: 60px;
+
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic font family in case the specific font you've chosen isn't available.